dwave/system/samplers/dwave_sampler.py

```diff
@@ -107,10 +107,6 @@ def __init__(self, config_file=None, profile=None, endpoint=None, token=None,
                                          endpoint=endpoint, token=token, proxy=proxy,
                                          permissive_ssl=permissive_ssl)
 
-        if solver_features is None:
-            # default to getting a QPU rather than a software solver
-            solver_features = {'qpu': True}
```

Comment:
Won't this override the dwave.conf optional property/value "client = "? That is, if it's not explicitly specified in the call but is specified in the dwave.conf configuration file, I'd expect the configuration value to hold, so if someone specifies "client = sw" that should be set

Reply:
Oh, sorry, just saw this is red, not green. After 8-hour meeting, I should go home